Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
```

Comment:
> Have you noticed that these functions are private? Because of progInfo they can't be internal. So overloads would too far a stretch.

Why would it be too far a stretch?

> At first I had progInfo as the second parameter. But I thought it to be more consistent when in every function that needs progInfo it has it as its first parameter.

Thinking about it, if we make them overloads it should be the third parameter.